mangum/protocols/http.py

```diff
@@ -50,6 +51,7 @@ class HTTPCycle:
     scope: Scope
     body: bytes
     text_mime_types: typing.List[str]
+    use_multi_value_headers: bool = True
```

Comment:
Does this need a default? I think it should always be passed through the adapter.